Lecture9-Understanding_and_Visualizing_Convolutional_Neural_Networks/README.md

```diff
@@ -1,12 +1,97 @@
+
+The simplest way to do this is to do the calculations `(V.T.dot(V))` after create a matrix `(= V)` with row 224x224 and column 64 by connecting all the fibers together.
+
+However, the **Gram Matrix** is not the most important thing that makes this possible. In fact, you can get similar results using the variants and mean of the activations instead of **Gram Matrix**.
+  
+**step 3**  
+Starting with a zero image, the content is optimized to be the same as the content target image, and the style is optimized to be the same as the style target image. In this optimization, we use only two images, so we can use **L-BFGS** (2nd order optimization) because all parameters fit in memory.
+
+## Can We Make ConvNet be Fool?
+We can pose an optimization over the input image to maximize any class score.  
+Q: Can we use this to "fool" ConvNets?  
+A: Yeah!
+
+So, for example, you can update any input image to maximize the ostrich score of ConvNet's classifier. Our expectation is that this process will transform the image into an image close to ostrich. However, it was possible to make the ostrich score very high just by applying a distortion that is so difficult for people to feel the change. You can start the same task entirely with random noise. An image that looks like random noise in the human eye can be classified as a specific class with a confidence level of 99.6% or more. This have happened before ConvNet. For example, in the case of a HOG, the HOG features of images can be made almost identical, even though images are completely different for the human. These are called **adversarial examples**.
+
+> primary cause of neural networks' vulnerability to adversarial perturbation is their **linear nature**.
+
+**Linear nature** of the functions that were used in the forward pass. Images are super high dimensional objects, lots of piexel so there is a hundred and fifty thousand dimensional space. And the real images that we actually trained on are statistical structures and they are constrained to a tiny manifold of this space. ConvNet were very well on that tiny manifold where statistics of images are image-like. But the outsides of the manifold were kind of casting shadows of complete randomness and chaos because we haven't trained there. 
+That is, after ConvNet is completed by searching for a very restrictive manifold in very high dimensional space, you will experience tremendous chaos in shadow that has not yet been searched (e.g. random noise).
+
+Also, since the input dimesions are so large that even very small element-wise variations are present, the sum is so large that the results are very severe. (fool binary linear classifier를 설명하긴 싫은데, 이 문장은 넣고 싶네요 ㅜㅜ 근데 뭔가 뜬금 없어 보여서 어떻게 해야 넣어야 할지 모르겠습니다.)
+
```

Comment:
Um.... I don't feel it sudden, but if you feel uncomfortable, then what about appending informative picture to implement the sentence?